src/signed-http-client.js

```diff
@@ -59,7 +59,21 @@ const get = async path => {
   return axios(signedRequest);
 };
 
+const remove = async path => {
```

Comment:
I think this should be called 'delete' since it'll make it consistent with the naming of the other

Reply:
Delete is reserved keyword, we can't name it that